modules/migration-configuring-proxy-for-dvm.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+// * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
+
+[id='migration-configuring-proxy-for-dvm_{context}']
+= Configuring a proxy for direct volume migration
+
+If you are performing direct volume migration from a source cluster behind a proxy, you must configure an Stunnel proxy in the `MigrationController` custom resource (CR). Stunnel creates a transparent tunnel between the source and target clusters for the TCP connection without changing the certificates.
+
+[NOTE]
+====
+Direct volume migration supports only one proxy. The source cluster cannot access the route of the target cluster if the target cluster is also behind a proxy.
+====
+
+.Prerequisites
+
+* You must have `cluster-admin` privileges on all clusters.
+
+.Procedure
+
+. Log in to the cluster on which the `migration-controller` pod runs.
+. Get the `MigrationController` CR manifest:
++
+[source,terminal]
+----
+$ oc get migration-controller <migration_controller> -n openshift-migration
+----
+
+. Add the `stunnel_tcp_proxy` parameter value to the manifest:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+...
+spec:
+  stunnel_tcp_proxy: <stunnel_proxy> <1>
+----
+<1> Specify the Stunnel proxy: `\http://<user_name>:<password>@<ip_address>:<port>`.
+
+. Save the manifest as `migration-controller.yaml`.
+. Apply the updated manifest:
++
+[source,terminal]
+----
+$ oc apply -f migration-controller.yaml -n openshift-migration
```

Comment:
it should use `replace` , such as 
`$ oc replace -f migration-controller.yaml -n openshift-migration`